_posts/2025-09-05-sqlserver-tx-log.adoc

```diff
@@ -0,0 +1,507 @@
+---
+layout: post
+title:  Peeking Inside the SQL Server Transaction Log
+date:   2025-08-29 00:00:00 -0000
+tags: [ sqlserver, tx-log, cdc-tables, internals]
+author: vjuranek
+---
+
+SQL Server offers change data capture (CDC) through link:https://learn.microsoft.com/en-us/sql/relational-databases/track-changes/about-change-data-capture-sql-server?view=sql-server-ver17[captured tables] - a special set of system tables that record modifications in selected “ordinary” tables.
```

Comment:
I think the correct term is change tables, not captured tables - see https://learn.microsoft.com/en-us/sql/relational-databases/track-changes/about-change-data-capture-sql-server?view=sql-server-ver17#change-table